src/SalmonDE/StatsPE/Entries/EntryManager.php

```diff
@@ -57,6 +57,6 @@ public function getEntry(string $entryName): ?Entry{
     }
 
     public function entryExists(string $entryName): bool{
-        return isset($this->entries[strtolower($entryName)]);
+        return isset(self::$entries[strtolower($entryName)]);
```

Comment:
`$this` = `EntryManager.php`, so shouldn't be different from doing `self::` instead. Also, you used `self::$entries[]` above, so keep consistency.